packages/cli/configuration/src/generators-yml/convertGeneratorsConfiguration.ts

```diff
@@ -242,6 +243,35 @@ async function parseAPIConfiguration(
     rawGeneratorsConfiguration: GeneratorsConfigurationSchema
 ): Promise<APIDefinition> {
     const apiConfiguration = rawGeneratorsConfiguration.api;
+
+    if (apiConfiguration != null && isApiConfigurationV2Schema(apiConfiguration)) {
+        return {
+            type: "singleNamespace",
```

Comment:
do we need to take into account the `namespace` configs to know if it's single or multi here?

Reply:
Yes we do, the entire APIDefinition data model needs to change so leaving as a follow up. Cohere needs this so will do shortly.